### DISCUSSION TITLE
Auditing PR's Merged by Bypassing Requirements

### DISCUSSION THREAD

**Discussion by cgamio** [community/community#109386] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We have a robust set of requirements (# of reviews + specific checks) required to merge PR's, but also some folks empowered to bypass those restrictions and merge in case of emergencies. Is there a way to identify PR's where those powers were exercised to merge a PR by bypassing requirements?

I've checked the API docs, and while PR's do have a "mergable" value, this gets nulled out once the PR is actually merged. There are also the check-runs, but those don't seem to identify which ones are required and they don't account for the case where the # of reviewers didn't meet the minimum.

**Comment by dserodio** [comment]
That would be very useful indeed (commenting so that it's not closed due to inactivity)

**Comment by bowtie-ltsa** [comment]
would love to have this also
